src/Components/Insights.js

```diff
@@ -0,0 +1,50 @@
+import '../Styles/Insights.css';
+
+import React, { Component } from 'react';
+import ReactApexChart from 'react-apexcharts';
+import { useState } from 'react';
+import Navbar from './Navbar';
+import { useRecoilValue } from 'recoil';
+import CuisineStats from '../recoils/CuisineStats';
+const Insights = () => {
+    const cuisines = useRecoilValue(CuisineStats);
+    const [options, setOptions] = useState({
+        chart: {
+            width: 380,
+            type: 'pie',
+        },
+        labels: ['Indian', 'Chinese', 'Mexican', 'Turkish'],
+        responsive: [{
+            breakpoint: 480,
+            options: {
+                chart: {
+                    width: 200
+                },
+                legend: {
+                    position: 'bottom'
+                }
+            }
+        }]
+    });
+    const [series, setSeries] = useState(cuisines);
+    let x = 0;
+    for (let i = 0; i < cuisines.length; i++) x += cuisines[i];
+    return (
+        <>
+            <Navbar />
```

Comment:
do not call Navbar from every component, instead place it in a common component.